mongo/database.go

```diff
@@ -350,24 +384,33 @@ func (db *Database) ListCollectionNames(ctx context.Context, filter interface{},
 	return names, nil
 }
 
-// ReadConcern returns the read concern of this database.
+// ReadConcern returns the read concern the database was configured with.
```

Comment:
nit, here and below: ending with preposition.  Also, capitalization, per my comment above.  How about  "ReadConcern returns the read concern used to configure the Database object"?

Reply:
Good idea. I was struggling to come up for a good way to word this. Changed for read concern, read pref, and write concern.